### PR TITLE
Project detail feature

### PR DESCRIPTION
**Pull request by shakyav**
#25 @gagansk 
1. added a dynamic route in the routes.tsx file for project details page
2. created a new file project_list_with_table.tsx which fetches the data from api with data fields as namespace and activation time
3. made changes in /myTable/Dashboartable/DashboardTable.tsx  to make the project names as hyperlinks, and added Active hours column
4. created a demoprojectfilterform.tsx which is a replica of ProjectFilterForm , only for reference. Once @nikunjdoshi67 integrates his code, we can remove this file
All the files that I created are located in src/app/project_page

Dashboard Screen 

image

once the user clicks on project links it will display the ProjectDetail(project_page/project_detail.tsx) component which @nikunjdoshi67 has to design

image

### CODE REVIEW

File: src/app/DataFilterForm/ProjectDataFilterForm.tsx [hunk 1]
```diff
@@ -1,154 +1,122 @@
...
+    );
 
-    }
+  }
 
 
 }
-export {ProjectDataFilterForm};
-
+export { ProjectDataFilterForm };
```
Comment (gagansk) [inline src/app/DataFilterForm/ProjectDataFilterForm.tsx]: Please add a newline here.

File: src/app/project_page/project_detail.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+   
+    return (
+
+        <PageSection>
+            <Title headingLevel="h1" size="lg">Projectpage</Title>
+            
+            <div>
+
+               {name}
```
Comment (gagansk) [inline src/app/project_page/project_detail.tsx]: Please add a text here called Name

File: src/app/project_page/project_detail.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+            </div>
+            <div>
+
+                {node}
```
Comment (gagansk) [inline src/app/project_page/project_detail.tsx]: Please add a text here called Node

File: src/app/project_page/project_detail.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+            </div>
+            <div>
+
+                {start}
```
Comment (gagansk) [inline src/app/project_page/project_detail.tsx]: Please add a text here called Start Time

File: src/app/project_page/project_detail.tsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+            </div>
+            <div>
+
+                {end}
```
Comment (gagansk) [inline src/app/project_page/project_detail.tsx]: Please add a text here called End time

File: src/app/project_page/project_detail.tsx [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+            </div>
+            <div>
+
+                {pod}
```
Comment (gagansk) [inline src/app/project_page/project_detail.tsx]: Please add a text here called Pod Name

File: src/app/project_page/project_detail.tsx [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+            </div>
+            <div>
+
+                {cpusage}
```
Comment (gagansk) [inline src/app/project_page/project_detail.tsx]: Please add a text here called CPU Utilization

File: src/app/project_page/project_detail.tsx [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+            </div>    
+           
+        </PageSection>
+
+
+
+    )
+}
+
+export { ProjectDetail };
```
Comment (gagansk) [inline src/app/project_page/project_detail.tsx]: Add a new line

**Review by gagansk — CHANGES_REQUESTED** [review]
Great work, @shakyav ! I have added some minor comments. 

Please address them and we can merge to master right after that.

File: src/app/project_page/project_detail.tsx (hunk 2 above)
Comment (shakyav) [reply src/app/project_page/project_detail.tsx]: @gagansk changes done

File: src/app/project_page/project_detail.tsx (hunk 3 above)
Comment (shakyav) [reply src/app/project_page/project_detail.tsx]: @gagansk changes done

File: src/app/project_page/project_detail.tsx (hunk 4 above)
Comment (shakyav) [reply src/app/project_page/project_detail.tsx]: @gagansk changes done

File: src/app/project_page/project_detail.tsx (hunk 5 above)
Comment (shakyav) [reply src/app/project_page/project_detail.tsx]: @gagansk changes done

File: src/app/project_page/project_detail.tsx (hunk 6 above)
Comment (shakyav) [reply src/app/project_page/project_detail.tsx]: @gagansk changes done

File: src/app/project_page/project_detail.tsx (hunk 7 above)
Comment (shakyav) [reply src/app/project_page/project_detail.tsx]: @gagansk changes done

File: src/app/project_page/project_detail.tsx (hunk 8 above)
Comment (shakyav) [reply src/app/project_page/project_detail.tsx]: @gagansk changes done

**Comment by gagansk** [comment]
LGTM.

Thanks!

**Pull request merged**